helm/thingsboard/values.yaml

```diff
@@ -82,6 +82,7 @@ node:
     port: 8080
   state:
     defaultInactivityTimeoutInSec: "600"
+    defaultStateCheckIntervalInSec: "60"
```

Comment:
What about thingaboard.transport.sessions.inactivity_timeout?

Reply:
I suggest to create a separate task to make sure we really need to touch this parameter. In the task we should check if inactive device is actually disconnected by thingsboard. I would act only after this. WDYT?

Reply:
Fine for me, but let's do the same then in the evp-chart PR

Reply:
@ricardnacher once the first version of the feature works e2e (hopefully tomorrow) I will try to reproduce the case you are talking about and see what happens. I will try to demo it tomorrow and we decide if and how we want to handle it. Does it make sense? Lets merge this for now, this is enough for the tomorrow demo.